src/appleseed-max-impl/appleseedrenderer/renderersettings.cpp

```diff
@@ -357,6 +353,7 @@ void RendererSettings::apply_settings_to_final_config(asr::Project& project) con
         params.insert_path("adaptive_tile_renderer.max_samples", m_adaptive_max_samples);
         params.insert_path("adaptive_tile_renderer.noise_threshold", m_adaptive_noise_threshold);
     }
+    params.insert_path("rendering_threads", m_rendering_threads);
```

Comment:
You need to preserve the logic that set the parameter to `auto` when it's 0 in Max.

Reply:
Yes, definitely true.